examples/rust/get_started/examples/11-attribute-based-authentication.rs

```diff
@@ -0,0 +1,91 @@
+/// INTRODUCTION
+///
+/// This example shows how to use attribute-based credentials in order to have
+/// several devices connecting to a server, via the Ockam Orchestrator.
+///
+/// The corresponding example using the command-line can be found here: https://docs.ockam.io/use-cases/apply-fine-grained-permissions-with-attribute-based-access-control-abac.
+///
+/// You first need to:
+///
+///  - create a project with `ockam enroll`
+///  - export the project information with `ockam project information > project.json`
+///
+/// Then you can start:
+///
+///  - a local Python webserver: `python3 -m http.server --bind 127.0.0.1 5000`
+///  - the control node in `11-attribute-based-authentication-control-plane.rs`
+///  - the edge node in `11-attribute-based-authentication-edge-plane.rs`
+///
+/// This will set up a TCP outlet on the control node, connected to the Python webserver
+/// and a TCP inlet on the edge node which can be used to send HTTP requests (at 127.0.0.1:7000).
+///
+/// Then if you execute `curl --fail --head --max-time 10 127.0.0.1:7000` you should get back
+/// a successful response like:
+///
+/// HTTP/1.0 200 OK
+/// Server: SimpleHTTP/0.6 Python/3.9.6
+/// Date: Tue, 07 Feb 2023 15:05:59 GMT
+/// Content-type: text/html; charset=utf-8
+/// Content-Length: 870
+///
+/// and observe that a successful connection has been made on the Python webserver:
+///
+/// ± python3 -m http.server --bind 127.0.0.1 5000                                                                                                                                                                                                                                                                                                                                                                                      default
+///  Serving HTTP on 127.0.0.1 port 5000 (http://127.0.0.1:5000/) ...
+///  127.0.0.1 - - [06/Feb/2023 15:52:20] "HEAD / HTTP/1.1" 200 -
+///
+/// TOPOLOGY
+///
+/// The network we establish between the control node, the edge node and the Orchestrator is the following
+///
+///                   +-------------------------+
+///                   | Edge node               | Inlet <-- 127.0.0.1:7000
+///                   |                         | connected to "outlet" via secure channel
+///                   +-------------------------+
+///                     |                 |
+///  secure channel to  |                 | create secure channel to control
+///  project            |                 | via the forwarder
+///                     v                 |
+///                   +-------------------+-------+                 +--------------+
+///                   | Orchestrator      |       | get credentials | Authority    |
+///                   |                   |       |---------------->|              |
```

Comment:
Aren't nodes talking to the Authority directly?

Reply:
good point, yes

Reply:
Fixed, thanks.